patsy/missing.py

```diff
@@ -183,7 +183,7 @@ def _handle_NA_drop(self, values, is_NAs, origins):
             total_mask |= is_NA
         good_mask = ~total_mask
         # "..." to handle 1- versus 2-dim indexing
-        return [v[good_mask, ...] for v in values]
+        return [v[good_mask] if v.ndim == 1 else v[good_mask, ...] for v in values]
```

Comment:
Is this due to upstream indexing changes? Kind of annoying if `...` no longer supports "zero" expansion.

Reply:
Yes, it appears that it is stricter and no longer supports expansion.  I only found this by running against the statsmodels test suite.

Reply:
Hmmm... this still works in my testing. Is there an upstream numpy change I should be worried about? (These are, or should be, numpy arrays still).